__tests__/port.tests.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+/* eslint-disable no-unused-vars */
```

Comment:
You should not have "unused" variables in your module, so probably not a good idea to turn this off.